lua/plenary/git.lua

```diff
@@ -0,0 +1,146 @@
+local Job = require 'plenary.job'
+
+local git = {}
+
+-- TODO(conni2461): What else? Need more ideas :)
+
+--- Creates a git commit table
+--- Requires format: --format=%h:%an:%aD:%s
+git.make_commit = function(commit_line)
```

Comment:
Might be better be private function, unless there is a use case for it outside of this module

Reply:
Haven't figured that one out yet :smile:. We will do that laster. Resolving for now